src/ert/scheduler/driver.py

```diff
@@ -61,6 +63,12 @@ async def poll(self) -> None:
     async def finish(self) -> None:
         """make sure that all the jobs / realizations are complete."""
 
+    def read_output_files(
+        self, runpath: str, job_name: str, num_tail_characters_to_read: int = 300
```

Comment:
Should we change `num_tail_characters_to_read` to `num_characters_to_read_from_end`?